### PR TITLE
Add sub-man rhsm to comps for pulpcore

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by ekohl** [comment]
Should we perhaps pick ca41b39217d0b38e6cd5e2e41b9eee0a7a9a669a? That also pulls in iniparse and I'm guessing that's a dependency. Since you made the PR, you might remember.

**Comment by jlsherrill** [comment]
good catch, cherry picked

File: rel-eng/tito.props [hunk 1]
```diff
@@ -1243,4 +1243,3 @@ disttag = .el8
 blacklist =
   python-iniparse
   subscription-manager
-
```
Comment (ekohl) [inline rel-eng/tito.props:1246]: Is this needed?

Comment (jlsherrill) [reply rel-eng/tito.props]: cherry pick conflict :)

**Pull request merged**